src/main/java/eod/warObject/character/concrete/red/MafiaAssassin.java

```diff
@@ -0,0 +1,77 @@
+package eod.warObject.character.concrete.red;
+
+import eod.Gameboard;
+import eod.Party;
+import eod.Player;
+import eod.card.abstraction.summon.SummonCard;
+import eod.card.concrete.summon.MafiaAssassinSummon;
+import eod.effect.RegionalAttack;
+import eod.exceptions.NotSupportedException;
+import eod.param.PointParam;
+import eod.warObject.Damageable;
+import eod.warObject.WarObject;
+import eod.warObject.character.abstraction.assaulter.Assassin;
+import eod.warObject.leader.Leader;
+
+import java.awt.*;
+import java.util.ArrayList;
+
+import static eod.effect.EffectFunctions.RequestRegionalAttack;
+
+public class MafiaAssassin extends Assassin {
+    public MafiaAssassin(Player player) {
+        super(player, 1, 3, Party.RED);
+    }
+
+    @Override
+    public SummonCard getSummonCard() {
+        SummonCard c = new MafiaAssassinSummon();
+        c.setPlayer(player);
+        return c;
+    }
+
+    @Override
+    public String getName() {
+        return "組織暗殺者";
+    }
+
+    @Override
+    public void attack() {
+        super.attack();
+        SpecialRegionalAttack a = (SpecialRegionalAttack) RequestRegionalAttack(player, attack).from(this);
+        a.to(getAttackRange());
```

Comment:
The cast will fail.
RequestRegionalAttack will return a RegionalAttack, not SpecialRegionalAttack in this class.

Reply:
fixed